src/SSHLibrary/pythonforward.py

```diff
@@ -1,9 +1,9 @@
 import select
 import socket
 import threading
-from robot.utils import platform
+from robot.utils import platform, PY2
 from .logger import logger
-if platform.PY2 and platform.WINDOWS:
+if PY2 and platform.WINDOWS:
```

Comment:
Also `WINDOWS` is available directly via `robot.utils`. Most likely it won't be removed from `robot.utils.platform`, but it would nevertheless be better to change the import to `from robot.utils import PY2, WINDOWS`.

Reply:
Done. Thanks!